### PR TITLE
A bot debug

### PR DESCRIPTION
**Pull request by hazeltorek**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -79,6 +79,10 @@ public void robotInit() {
     SmartDashboard.putString("Git Branch", branch);
     SmartDashboard.putString("Git Commit", commit);
 
+    System.out.println((checkType() == RobotType.A_BOT) ? "A Robot" : "B Robot");
```
Comment (20MatthewNeal) [inline src/main/java/frc/robot/Robot.java]: Can we have this on shuffleboard

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 2]
```diff
@@ -26,6 +26,8 @@
   private double currentAngleLeft;
   private double currentAngleRight;
 
+  // of course theyre different
+  private double intakeSpeed = INTAKE_SPEED;
```
Comment (20MatthewNeal) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: . . . . better documentation

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 3]
```diff
@@ -37,10 +39,17 @@ public IntakeSubsystem() {
...
-    } else {
-      deployRight.setInverted(true);
     }
 
+    // why wouldnt they be different ?????
+    if (Robot.checkType() == Robot.RobotType.B_BOT) {
+      // differnet gear ratios
+      // because of course they are
+      intakeSpeed = 0.9;
+    }
```
Comment (20MatthewNeal) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: ?

Comment (hazeltorek) [reply src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: ???

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 4]
```diff
@@ -54,16 +60,26 @@ public IntakeSubsystem() {
...
+
+  public void resetEncoders() {
+    deployLeft.getSensorCollection().setQuadraturePosition(0, 0);
+    deployRight.getSensorCollection().setQuadraturePosition(0, 0);
   }
 
   /** 
   * Deploys the intake device for picking up cargo
   */
   public void deployIntake()
```
Comment (20MatthewNeal) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: Go into more detail on how this works i.e., scales the difference of the two encoders to ensure that they move at the same speed.

**Review by 20MatthewNeal — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java (hunk 4 above)
Comment (hazeltorek) [reply src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: to the nth

**Review by VZamora5 — APPROVED** [review]

**Review by 20MatthewNeal — APPROVED** [review]

**Pull request merged**